service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -177,7 +178,7 @@ public RegistryManager(String hostName, TokenCredential credential)
      */
     public RegistryManager(String hostName, TokenCredential credential, RegistryManagerOptions options)
     {
-        Objects.requireNonNull(credential, "credential cannot be null");
+        Objects.requireNonNull(credential, "credentialCache cannot be null");
```

Comment:
This gets thrown back to the user. So it should still read 'credential cannot be null' as they are not aware of our internal implementation.